hw/ip/rstmgr/doc/_index.md

```diff
@@ -170,24 +156,20 @@ The reset manager then checks as follows:
 
 ## Hardware Interfaces
 
-{{< incGenFromIpDesc "/hw/top_earlgrey/ip/rstmgr/data/autogen/rstmgr.hjson" "hwcfg" >}}
+### Parameters
+
+The following table lists the instantiation parameters of `rstmgr`.
+
+
+Parameter                   | Default       | Description
+----------------------------|---------------|---------------
+`SecCheck`                  | 1             | Enables reset consistency checks on the leaf reset.  Each check contains a small FSM
```

Comment:
nitnit: missing period at the end of the sentence

Reply:
fixed.